src/components/Table/Sorting.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Header, RowData, SortDirection } from "@tanstack/react-table";
+import { PropsWithChildren } from "react";
+import { ArrowUp, ArrowDown } from "./Icons";
+
+type Props = {
+	direction: SortDirection | false;
+};
+
+function SortingArrow({ direction }: Props) {
+	if (direction === false) return null;
+
+	const Comp = direction === "asc" ? ArrowUp : ArrowDown;
+
+	return <Comp className="h-4 w-4 shrink-0" />;
```

Comment:
If this is a proper svg we could aslo use `height={16} width={16}`. More of a suggestion/idea though

Reply:
just extracted it from pro. i think there's some weird cases with the height/width directly, i'd prefer to keep this way